app/electron/preload.js

```diff
@@ -15,8 +14,7 @@ contextBridge.exposeInMainWorld("api", {
   i18nextElectronBackend: i18nextBackend.preloadBindings(ipcRenderer),
   store: store.preloadBindings(ipcRenderer, fs),
   contextMenu: ContextMenu.preloadBindings(ipcRenderer),
-  //  send: name => sendHello(name),
-  //  get: ()=> getCollege(),
+  api: events.preloadBindings(ipcRenderer, fs),
```

Comment:
Check if we can give this name better than "api", as this object is already going to be in api, so the call from frontend will be window.api.api.something()